--- a/node.gyp	2020-11-04 15:55:48.000000000 +0100
+++ b/node.gyp	2020-11-04 15:55:51.000000000 +0100
@@ -1467,6 +1467,16 @@
         }],
       ],
     }, # cctest
+    {
+      'target_name': 'libv8_monolith',
+      'type': 'none',
+      'includes': [
+        'node.gypi'
+      ],
+      #'dependencies': [
+      #  'deps/v8/gypfiles/v8.gyp:v8_monolith',
+      #],
+    },
   ], # end targets
 
   'conditions': [
--- a/deps/v8/gypfiles/v8.gyp	2020-11-04 16:34:06.000000000 +0100
+++ b/deps/v8/gypfiles/v8.gyp	2020-11-04 16:34:10.000000000 +0100
@@ -1726,5 +1726,15 @@
     {
       'includes': [ 'v8_external_snapshot.gypi' ],
     }, # v8_external_snapshot
+    {
+      'target_name': 'v8_monolith',
+      'type': 'static_library',
+      'dependencies': [
+        'v8',
+        'v8_libbase',
+        'v8_libplatform',
+        'v8_libsampler',
+      ],
+    }
   ],
 }

